packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/provision.result.v2.bicep

```diff
@@ -3,6 +3,7 @@ module simpleAuthProvision './simpleAuthProvision.result.v2.bicep' = {
   name: 'simpleAuthProvision'
   params: {
     provisionParameters: provisionParameters
+    userAssignedIdentityId: 
```

Comment:
it should be 
userAssignedIdentityId: userAssignedIdentityProvision.outputs.identityResourceId?

Reply:
this test case only test simpleauth without identity, later will update the test case.